### PR TITLE
New Daily Digest Subject

### LINKED ISSUE #7186: Daily Digest Subject Cleanup
**Describe the bug**

Currently, this is an example of a subject from the Daily Digest:

`💬 63 Chats, 📣 1 Announcement, 💰44 Bounties, 🛠📆5 Events, 🎯1 Quest, ⚡️ 1 New Action, 👤1 Trending Avatar`

**To Reproduce**

Subscribe to the Daily Digest and or check-in Sendgrid.

**Expected behavior**

**Best practices:**

* Subject line to be less than 9 words
* We suggest using no more than 1 Emoji
* Keep your character count below 60
* No more than 3 punctuation marks 

**Screenshots**

image

**Desktop (please complete the following information):**

- OS: macOS
- Browser: Chrome 
- Browser Version: Version 84.0.4147.89 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

n/a

**Additional context**

I'm not sure what should be trimmed. We definitely want to highlight dynamic content like _XX Chats_. Would love to get @owocki's input.

### PR DESCRIPTION
**Pull request by molecula451**
##### Description

This PR will purge: 

🛠📆 Events
💬  Chats, 
📣 Announcement,

on the daily subject

And it'll keep it simple with 💰Bounties, 🎯 Quest, ⚡️ New Action, & More (including quantities of each) 

so it complies with best practices for marketing purposes proposed by @jdorfman 

##### Refers/Fixes

fixes #7186 

##### Testing

debug

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

**Review by jdorfman — CHANGES_REQUESTED** [review]
Almost there, need a space after each emoji e.g. `✅ Foo` not `✅Bar`. In the screenshot, the emoji is touching.

**Comment by molecula451** [comment]
> Almost there, need a space after each emoji e.g. `✅ Foo` not `✅Foo`. In the screenshot, the emoji is touching.

spaced in https://github.com/gitcoinco/web/pull/7329/commits/24e1852a3290dbb7730aad1ca83194fd488bf41e

**Review by jdorfman — APPROVED** [review]
🚀

**Comment by jdorfman** [comment]
@mehul0698 once this gets merged I will release the funds.

**Comment by molecula451** [comment]
> @mehul0698 once this gets merged I will release the funds.

@jdorfman i think you mean @molecula451 😆 push it @thelostone-mc :)

**Comment by jdorfman** [comment]
> @jdorfman i think you mean @molecula451 😆 push it @thelostone-mc :)

🤦‍♂️ yes, I meant that.

**Comment by molecula451** [comment]
@thelostone-mc push it

**Comment by molecula451** [comment]
fixed checks now passing good to push @thelostone-mc

**Review by octavioamu — APPROVED** [review]

File: app/marketing/mails.py [hunk 1]
```diff
@@ -1237,34 +1237,13 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None, featured_bounties=[
...
-            if chats_count:
-                plural = 's' if chats_count > 1 else ''
-                chat = f"💬 {chats_count} Chat{plural}"
-
-        notifications = get_notification_count(profile, 7, timezone.now())
-        if notifications:
-            plural = 's' if notifications > 1 else ''
-            notifications = f"🔵 {notifications} Notification{plural}"
-        else:
-            notifications = ''
```
Comment (octavioamu) [inline app/marketing/mails.py:1267]: this will not remove notification email from townsquare?

Comment (thelostone-mc) [reply app/marketing/mails.py:1267]: @jdorfman

Comment (molecula451) [reply app/marketing/mails.py:1267]: This PR only take care of the subject, by keeping it clean only using 💰Bounties, 🎯 Quest, ⚡️ New Action, & More (including quantities of each). So yeah the notifications subject is being taken out here

Comment (jdorfman) [reply app/marketing/mails.py]: @octavioamu @thelostone-mc this is for the Daily Digest. Town Square is in the email but removed from the subject. This was approved by @owocki

**Comment by molecula451** [comment]
Closing this out as there were a lot of changes recently on the mail.py https://github.com/gitcoinco/web/commit/708165ec9597bb5ed1d629ddf992c06e34a0b61e  and this branch it behind will have to open a clean branch

**Pull request closed without merging**